Tiled/extensions/diablo-dun.js

```diff
@@ -252,7 +260,6 @@ var dunMapFormat = {
 		var file = new BinaryFile(fileName, BinaryFile.WriteOnly);
 		file.write(buffer);
 		file.commit();
-		file.close();
```

Comment:
Note: I didn't realize in my last PR that `file.commit()` already closes the file. Apparently, I didn't try the `Save` button to test this, but I have tested it now.